insights/parsers/kdump.py

```diff
@@ -205,19 +206,37 @@ def using_local_disk(self):
         """
         Is kdump configured to only use local disk?
 
-        The logic used here is the first of these conditions:
+        More than one dump targets will lead to kudmp service start failure.
+        So let's suppose that only one target is set here.
+
+        The logic used here is:
 
         * If 'raw' is given as an option, then the dump is local.
         * If 'ssh', 'net', 'nfs', or 'nfs4' is given, then the dump is NOT local.
+        * If '<fs type> <partition>' is given, then the dump is local.
```

Comment:
Don't you want to implement this logic here? but in fact its in `is_using_local_fs`